ext/suse/pxp-agent.init

```diff
@@ -85,8 +85,21 @@ case "$1" in
         ## Stop daemon with killproc(8) and if this fails
         ## set echo the echo return value.
 
-        killproc -QUIT -p "${pidfile}" "${exec}" && rm -f "${lockfile}" "${pidfile}"
+        ## This accounts for the behavior of killproc on sles 11.
+        ## Check to see if a running process matches the contents
+        ## of the pidfile, and if so kill it. This could fail if
+        ## multiple pxp-agent processes are running, in which case
+        ## killproc may succeed or manual intervention is required.
```

Comment:
Yeah, I had more in the commit message but makes sense to put it in the comment.